packages/dx-react-grid/src/plugins/virtual-table/virtual-table-state.tsx

```diff
@@ -34,18 +34,22 @@ class VirtualTableStateBase extends React.PureComponent<VirtualTableStateProps,
 
   requestNextPageAction = (
     { referenceIndex, forceReload },
-    { virtualRows, skip }: Getters,
+    { virtualRows }: Getters,
   ) => {
     const { pageSize, totalRowCount } = this.props;
 
     let newBounds;
     let requestedRange;
     let actualVirtualRows = virtualRows;
+    let loadedInterval;
     if (forceReload) {
-      newBounds = requestedRange = { start: skip, end: skip + pageSize! * 2 };
+      loadedInterval = intervalUtil.getRowsInterval(virtualRows);
+      newBounds = requestedRange = getForceReloadInterval(
```

Comment:
Why we are using this notation `a = b = value` and are not using `let a, b, c;`?

I think these things are the same. We should not use `a = b = value`